dashboard/app/models/script.rb

```diff
@@ -333,8 +333,10 @@ def self.lesson_extras_script_ids
     @@lesson_extras_script_ids ||= all_scripts.select(&:lesson_extras_available?).pluck(:id)
   end
 
-  def self.maker_units
-    @@maker_units ||= visible_units.select(&:is_maker_unit?)
+  def self.maker_units(user)
+    total_units = visible_units
+    total_units += all_scripts.select {|s| s.has_pilot_access?(user)}
```

Comment:
Do you know if this line will cause any db queries (once all_scripts is cached)?  It seems a bit expensive to me to have to iterate through all scripts (but we probably do this elsewhere too).

Reply:
Thanks for chatting with me about this- I've made a ticket for our backlog to fix this in all places: https://codedotorg.atlassian.net/browse/LP-2228